### PR TITLE
fix 27116 - reload is not needed anymore on change of orientation/scr…

### PR DESCRIPTION
**Pull request by nhaagen**
https://mantis.ilias.de/view.php?id=27116

### CODE REVIEW

**Review by Amstutz — APPROVED** [review]
Hurray! This was one of my biggest pain points, glad to see it gone.

Just one question:

- [ ] Outcommented Code: You added two lines of outcommented code, why? Maybe get rid of it. However, they somehow look like they might be importantn in the future.

@klees merge as soon as i.O..

File: src/UI/templates/js/Page/stdpage.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ il.UI = il.UI || {};
 
 		var breakpoint_max_width = 768, //this corresponds to @grid-float-breakpoint-max, see mainbar.less/metabar.less
 			mq_orientation = window.matchMedia("(orientation: portrait)");
+			//attach orientation-change listeners directly to mediaquery:
+			//*mq_orientation.addListener(function(mq) {...});
```
Comment (klees) [inline src/UI/templates/js/Page/stdpage.js]: Just remove this, please.

Comment (klees) [reply src/UI/templates/js/Page/stdpage.js]: Should have read @Amstutz' comment first... =)

**Comment by Amstutz** [comment]
Thx a lot @nhaagen , I just resolved conflicts, merged manually forwarded to 6.

**Pull request merged**